jenkins/test-history/gen_html.py

```diff
@@ -46,19 +46,26 @@
 ])
 
 
+BUCKET_TO_PREFIX = {}
 PREFIX_TO_URL = {}
 
-def prefix_for(prefix):
-    if 'azure' in prefix:
-        new = 'azure$'
-    elif 'rktnetes' in prefix:
-        new = 'rktnetes$'
-    elif 'gs://kubernetes-jenkins/' in prefix:
-        new = ''
-    else:
-        new = prefix[5:-1].replace('/', '_') + '$'
-    PREFIX_TO_URL[new] = prefix
-    return new
+
+def load_buckets_info(infile):
+    BUCKET_TO_PREFIX.clear()
+    if infile is None:
```

Comment:
I'm going to make the "--buckets" argument required.  It simplifies the logic, and matches the pattern of other file arguments, such as "--input" for the tests.json file.